src/FSharp.Core/list.fsi

```diff
@@ -1634,6 +1825,9 @@ module List =
 
     /// <summary>Splits the collection into two collections, containing the 
     /// elements for which the given predicate returns True and False
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(n)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence